liquibase-core/src/main/java/liquibase/parser/core/formattedsql/FormattedSqlChangeLogParser.java

```diff
@@ -26,6 +26,155 @@
+
+    private static final String RUN_IN_TRANSACTION_REGEX = ".*runInTransaction:(\\w+).*";
+    private static final Pattern RUN_IN_TRANSACTION_PATTERN = Pattern.compile(RUN_IN_TRANSACTION_REGEX, Pattern.CASE_INSENSITIVE);
+
+    private static final String DBMS_REGEX = ".*dbms:([^,][\\w!,]+).*";
+    private static final Pattern DBMS_PATTERN = Pattern.compile(DBMS_REGEX, Pattern.CASE_INSENSITIVE);
+
+    private static final String IGNORE_REGEX = ".*ignore:(\\w*).*";
+    private static final Pattern IGNORE_PATTERN = Pattern.compile(IGNORE_REGEX, Pattern.CASE_INSENSITIVE);
+
+    private static final String FAIL_ON_ERROR_REGEX = ".*failOnError:(\\w+).*";
+    private static final Pattern FAIL_ON_ERROR_PATTERN = Pattern.compile(FAIL_ON_ERROR_REGEX, Pattern.CASE_INSENSITIVE);
+
+    private static final String ON_FAIL_REGEX = ".*onFail:(\\w+).*";
+    private static final Pattern ON_FAIL_PATTERN = Pattern.compile(ON_FAIL_REGEX, Pattern.CASE_INSENSITIVE);
+
+    private static final String ON_ERROR_REGEX = ".*onError:(\\w+).*";
+    private static final Pattern ON_ERROR_PATTERN = Pattern.compile(ON_ERROR_REGEX, Pattern.CASE_INSENSITIVE);
+
+    private static final String ON_UPDATE_SQL_REGEX = ".*onUpdateSQL:(\\w+).*";
+    private static final Pattern ON_UPDATE_SQL_PATTERN = Pattern.compile(ON_UPDATE_SQL_REGEX, Pattern.CASE_INSENSITIVE);
+
+    private static final String ON_SQL_OUTPUT_REGEX = ".*onSqlOutput:(\\w+).*";
+    private static final Pattern ON_SQL_OUTPUT_PATTERN = Pattern.compile(ON_SQL_OUTPUT_REGEX, Pattern.CASE_INSENSITIVE);
+
+    private static final String ROLLBACK_CHANGE_SET_ID_REGEX = ".*changeSetId:(\\S+).*";
+    private static final Pattern ROLLBACK_CHANGE_SET_ID_PATTERN = Pattern.compile(ROLLBACK_CHANGE_SET_ID_REGEX, Pattern.CASE_INSENSITIVE);
+
+    private static final String ROLLBACK_CHANGE_SET_AUTHOR_REGEX = ".*changesetAuthor:(\\S+).*";
+    private static final Pattern ROLLBACK_CHANGE_SET_AUTHOR_PATTERN = Pattern.compile(ROLLBACK_CHANGE_SET_AUTHOR_REGEX, Pattern.CASE_INSENSITIVE);
+
+    private static final String ROLLBACK_CHANGE_SET_PATH_REGEX = ".*changesetPath:(\\S+).*";
+    private static final Pattern ROLLBACK_CHANGE_SET_PATH_PATTERN = Pattern.compile(ROLLBACK_CHANGE_SET_PATH_REGEX, Pattern.CASE_INSENSITIVE);
+
+    private static final String ROLLBACK_MULTI_LINE_START_REGEX = "\\s*\\/\\*\\s*liquibase\\s*rollback\\s*$";
+    private static final Pattern ROLLBACK_MULTI_LINE_START_PATTERN = Pattern.compile(ROLLBACK_MULTI_LINE_START_REGEX, Pattern.CASE_INSENSITIVE);
+
+
+    private static final String WORD_RESULT_REGEX = "^(?:expectedResult:)?(\\w+) (.*)";
+    private static final String SINGLE_QUOTE_RESULT_REGEX = "^(?:expectedResult:)?'([^']+)' (.*)";
+    private static final String DOUBLE_QUOTE_RESULT_REGEX = "^(?:expectedResult:)?\"([^\"]+)\" (.*)";
+
+    private static final Pattern[] PATTERNS = new Pattern[]{
```

Comment:
I see the Pattern[] was named `patterns` in the previous code. That wasn't very confusing because `patterns` was declared where it was being used. Now that it is moved to a different location, a more descriptive name would be useful. Perhaps something like WORD_AND_QUOTING_PATTERNS?

Reply:
Done